Example/components/TabView.js

```diff
@@ -44,6 +44,7 @@ class TabView extends React.Component {
         <Button onPress={() => { Actions.tab_4(); }}>Switch to tab4</Button>
         <Button onPress={() => { Actions.tab_5({ data: 'test!' }); }}>Switch to tab5 with data</Button>
         <Button onPress={() => { Actions.echo(); }}>push clone scene (EchoView)</Button>
+        <Button onPress={() => { Actions.echo(); }}>push clone scene (EchoView)</Button>
```

Comment:
What is this for?

Reply:
Duplicated line, can be removed. My mistake.

Reply:
Please, thanks! it will reduce confusion for novice react native devs.

Reply:
Do you want me to remove the line and push the changes?

Reply:
Yes please

Reply:
Done